DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/vlan/test_vlan_basic_functionality.py

```diff
@@ -0,0 +1,91 @@
+import asyncio
+
+import pytest
+from dent_os_testbed.utils.test_utils.tgen_utils import (tgen_utils_get_dent_devices_with_tgen,
+                                                         tgen_utils_setup_streams, tgen_utils_start_traffic,
+                                                         tgen_utils_stop_traffic, tgen_utils_get_traffic_stats,
+                                                         tgen_utils_get_loss, tgen_utils_dev_groups_from_config,
+                                                         tgen_utils_traffic_generator_connect, )
+from dent_os_testbed.utils.test_utils.br_utils import (configure_bridge_setup, configure_vlan_setup,
+                                                       get_traffic_port_vlan_mapping)
+
+
+pytestmark = [pytest.mark.suite_functional_vlan,
+              pytest.mark.asyncio,
+              pytest.mark.usefixtures("cleanup_bridges", "cleanup_tgen")]
+
+port_map = ({"port": 0, "settings": [{"vlan": 1, "untagged": False, "pvid": False}]},
+            {"port": 1, "settings": [{"vlan": 1, "untagged": False, "pvid": False}]},
+            {"port": 2, "settings": [{"vlan": 1, "untagged": False, "pvid": False}]},
+            {"port": 3, "settings": [{"vlan": 2, "untagged": False, "pvid": False}]})
+
+
+async def test_vlan_basic_functionality(testbed):
+    """
+    Test Name: VLAN basic functionality
+    Test Suite: suite_functional_vlan
+    Test Overview: Test VLAN basic functionality with broadcast packet forwarding
+    Test Procedure:
+    1. Initiate test params.
```

Comment:
Please describe at least briefly what you configure here. This description doesn't let anyone to understand test purpose. The same is about other steps: traffic profiles, and expectations.
Please apply this to all tests.